agent/command/results_utils.go

```diff
@@ -63,22 +62,13 @@ func sendTestLogsAndResults(ctx context.Context, comm client.Communicator, logge
 }
 
 func attachTestResults(ctx context.Context, conf *internal.TaskConfig, td client.TaskData, comm client.Communicator, results []testresult.TestResult) error {
-	var failed bool
-	var err error
 	output, ok := conf.Task.GetTaskOutputSafe()
-	// Version 0 denotes that the task will use cedar's test result service,
-	// whereas other version numbers indicate that the task uses evergreen's
-	// test result service.
-	if !ok || output.TestResults.Version == 0 {
-		failed, err = uploadTestResultsCedar(ctx, comm, conf, results)
-		if err != nil {
-			return errors.Wrap(err, "attaching test results to Cedar")
-		}
-	} else {
-		failed, err = uploadTestResults(ctx, comm, conf, results, td, output)
-		if err != nil {
-			return errors.Wrap(err, "attaching test results")
-		}
+	if !ok {
```

Comment:
Should this check the test results version just to be sure? Or `switch` case on it?